e2e/test/prerequisites/E2ETestsSetup/e2eTestsArmTemplate.json

```diff
@@ -255,9 +255,14 @@
     },
     {
       "type": "Microsoft.Devices/IotHubs",
-      "apiVersion": "2019-11-04",
+      "apiVersion": "2020-01-01",
       "name": "[parameters('HubName')]",
       "location": "[resourceGroup().location]",
+      "identity": {
```

Comment:
This will create a system assigned identity to the IOT hub. All fields must be left blank.